tests/filesystem.go

```diff
@@ -155,6 +155,9 @@ func runIgnition(t *testing.T, ctx context.Context, stage, root, cwd string, app
 	cmd.Dir = cwd
 	cmd.Env = append(os.Environ(), appendEnv...)
 	out, err := cmd.CombinedOutput()
+	if _, ok := err.(*exec.ExitError); err != nil && !ok {
```

Comment:
Hm, why specifically `ExitError`?  What if it was somehow not in `$PATH`?  IOW why not just the usual:

```
if err != nil {
  return fmt.Errorf("failed to run Ignition: %v, err)
}
```?

Reply:
Oh sorry I read too quickly, this function is expecting to return errors.  You're testing for *not* `ExitError`.